cf-check/diagnose.c

```diff
@@ -316,19 +318,15 @@ static int fork_and_diagnose(const char *path)
     return CF_CHECK_OK;
 }
 
-int diagnose_main(int argc, char **argv)
+int diagnose_files(Seq* filenames)
```

Comment:
Could/should return `size_t` now, right?

Reply:
Fixed.